### DISCUSSION TITLE
Problem with connecting to Cartridge Tarantool

### DISCUSSION THREAD

**Discussion by drozdoff-dev** [tarantool/tarantool#7665] [Q&A] [answered]
Good afternoon, I ran into a problem that the Cartridge runs like without problems, but with one error, which they say does not affect anything. The screenshot is shown below.
![Снимок экрана от 2022-09-10 06-52-54](https://user-images.githubusercontent.com/109925730/189467620-a7af77f2-18a9-4002-b999-7eca4db4ad36.png)
But the main problem is that I need to connect to the Cartridge from a python or node js application, for example, I took python and when connecting with guest, the error "Execute access to function 'crud.select' is denied for user 'guest'" is shown, that is, I can't use CRUD(Cartridge), when connecting from the administrator, I do not know the password, also when connecting from a guest, there is an error "Execute access to universe " is denied for user 'guest'".
Please help either allow login with guest, or specify the administrator password, or create a user with all the necessary permissions.

**Comment by Totktonada** [comment, marked as answer]
*(Answered in #7666, will cite here.)*

Give all right to the guest user:

```lua
box.schema.user.grant('guest', 'super')
```

You can find description of all the user management functions in the [documentation](https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_schema/).
